### DISCUSSION TITLE
Where can I find community-made designs?

### DISCUSSION THREAD

**Discussion by nlguerra2003** [JanDeDobbeleer/oh-my-posh#928] [Troubleshoot]
None of the default designs are good enough for my liking and I'm too lazy to be thinking how to improve them plus tweaking their values. Is there any website where people share their designs? I've seen websites to share Windows Terminal color schemes, so this wouldn't be too odd to me.

**Comment by JanDeDobbeleer** [comment]
Not that I'm aware off.
